src/test/java/pages/DominoPage.java

```diff
@@ -0,0 +1,87 @@
+package pages;
+
+import elements.*;
+import indices.Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import properties.Property;
+import util.ScrollUtils;
+import util.WaitUtils;
+
+
+public class DominoPage extends BasePage {
+
+    public static final String DOMINO_ENDPOINT = "dominos";
+
+    @FindBy(xpath = "//input[@placeholder='от']")
+    protected Input minPriceInput;
+
+    @FindBy(xpath = "//*[@id='schema-products']/div/nobr")
+    protected Text warningMessage;
+
+    @FindBy(className = "schema-header__title")
+    protected Text headerLabel;
+
+    @FindBy(xpath = "//*[@id='schema-products']/div[1]/descendant::div[2]")
+    protected Picture pizzaPic;
+
+    public DominoPage() {
+        super();
+    }
+
+    @Override
+    public DominoPage openPage() {
+        driver.navigate().to(props.getKeyProperty(Property.valueOf(Property.CATALOGUE_URL + DOMINO_ENDPOINT)));
+        logger.debug("Navigation to the catalogueURL..." + DOMINO_ENDPOINT);
```

Comment:
It's better to make all the similar logs with same messages (it makes it easier to look through such logs). So I'd recommend to update log message to "Navigation to the URL" in all the similar cases

Reply:
ok